js/test/.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
       "curly": "off",
       "rest-spread-spacing": "off",
       "no-multiple-empty-lines": "off",
+      "no-undef": "off",
```

Comment:
It would be better to achieve the same effect by tuning environment (i.e. without relaxing the checks):
```
"env": {
    "node": true,
    "jasmine": true,
  }
```

Reply:
@mkutny fixed: https://github.com/ccxt/ccxt/pull/1635/commits/47c9d5a8a3d7a16c49e472c83c15703b9aa15a86

Reply:
@xpl but the linter config is still broken because of `"indent": ["error", "space"]` rule. Does it work for you?

Reply:
@mkutny yep, it was broken (wrong syntax), thanks for pointing out. Replaced with `"indent": ["warn", 4, { "ignoredNodes": ["ObjectPattern"] }],` in `base/.eslintrc` and `tests/base/.eslintrc`.

I use "weird" indentation in destructuring patterns sometimes, and ESLint seems just isn't flexible enough to formally describe that. As we do not transpile the `base` code yet (but we will, eventually) – it shouldn't be a problem for now.